.gitignore

```diff
@@ -7,6 +7,5 @@ spec/dummy/tmp/
 .rvmrc
 .idea
 coverage/
- 
-Gemfile.lock
 
+Gemfile.lock
```

Comment:
It's pretty common for gems to exclude the `Gemfile.lock`.  I think it's because it allows Travis to automatically use the latest versions, which may detect bugs early.